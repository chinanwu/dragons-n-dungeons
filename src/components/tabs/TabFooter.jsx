import React from 'react';
import PropTypes from 'prop-types';

// TODO: still working on the name
export const TabFooter = ({ children }) => {
  return children;
};

TabFooter.propTypes = {
  children: PropTypes.any,
};

export default TabFooter;
