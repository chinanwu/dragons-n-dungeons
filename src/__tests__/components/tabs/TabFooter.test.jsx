import { shallow } from 'enzyme';
import React from 'react';

import { TabFooter } from '../../../components/tabs/TabFooter.jsx';

jest.unmock('../../../components/tabs/TabFooter.jsx');

describe('TabFooter component', () => {
  describe('rendering', () => {
    it('renders TabFooter', () => {
      const wrapper = shallow(<TabFooter />);
      expect(wrapper.getElement()).toMatchSnapshot();
    });
  });
});
