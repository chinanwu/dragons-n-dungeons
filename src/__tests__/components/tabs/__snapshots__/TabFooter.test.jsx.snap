// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabFooter component rendering renders TabFooter 1`] = `null`;
